### PR TITLE
Added management command to populate initial history as of 6/29.

### PR DESCRIPTION
**Pull request by HassanJaveed84**

### CODE REVIEW

File: course_discovery/apps/course_metadata/management/commands/backfill_history.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                    values = [[row[column.upper()] for column in columns] for row in rows]
+
+                    # Replace 'NULL' with None
+                    values = self.replace_values(values, 'NULL', None)
+                    # Replace 'true' with True
+                    values = self.replace_values(values, 'true', True)
+                    # Replace 'false' with False
+                    values = self.replace_values(values, 'false', False)
+                    # Add history columns data
+                    for value in values:
```
Comment (pwnage101) [inline course_discovery/apps/course_metadata/management/commands/backfill_history.py]: I know this is just a one-off command, but I'll just say it anyway:
```suggestion
                    for row in values:
```

File: course_discovery/apps/course_metadata/management/commands/backfill_history.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+                        value.extend([history_date, self.HISTORY_CHANGE_REASON, '+', self.HISTORY_USER_ID])
+                    # Convert to tuple
+                    values = [tuple(value) for value in values]
```
Comment (pwnage101) [inline course_discovery/apps/course_metadata/management/commands/backfill_history.py]: ditto

File: course_discovery/apps/course_metadata/management/commands/backfill_history.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+            action="store",
+            help="Path containing data files from snapshot for history backfill"
+        )
+
+    def chunks(self, ids, chunk_size):
+        for i in range(0, len(ids), chunk_size):
+            yield ids[i:i + chunk_size]
+
+    def replace_values(self, values, original, replacement):
+        values = [[replacement if v == original else v for v in value] for value in values]
```
Comment (pwnage101) [inline course_discovery/apps/course_metadata/management/commands/backfill_history.py]: ditto

**Review by pwnage101 — APPROVED** [review]
LGTM, just a nit that's particularly annoying to me.

**Review by estute — APPROVED** [review]
I fixed up the nits you mentioned, @pwnage101

**Review by pwnage101 — APPROVED** [review]

**Review by doctoryes — APPROVED** [review]

**Review by Dillon-Dumesnil — APPROVED** [review]

**Pull request merged**